Sources/WishKit/View/WishCell.swift

```diff
@@ -30,7 +30,13 @@ final class WishCell: UITableViewCell {
 
     private let badgeView = BadgeView()
 
-    private let descriptionLabel = UILabel(font: .systemFont(ofSize: 13), lineCount: WishKit.config.expandDescriptionInList ? 0 : 1)
+    private var descriptionLabel = UILabel(font: .systemFont(ofSize: 13), lineCount: WishKit.config.expandDescriptionInList ? 0 : 1)
+
+    public var isExpanded = WishKit.config.expandDescriptionInList {
```

Comment:
Does this need to be a stored property or can it be computed?

Reply:
This Boolean can be toggled to expand/shrink the `WishCell` by tapping on it. So I don't think it can be computed.